app/gui/src/project-view/util/codemirror/language/index.ts

```diff
@@ -60,23 +51,9 @@ const completionBindings = keymap.of([
 ])
 
 /** @returns a reactive syntax support extension for the specified language. */
-export function useLanguageSupport(
-  syntax: ToValue<Opt<string>>,
-  { project, projectNames, suggestionDb }: LanguageSupportOptions,
-) {
-  const extensions: Record<string, Readonly<Ref<Extension>>> = Object.assign(Object.create(null), {
-    'enso-table-expression': useTableExpressionExtension({
-      project,
-      projectNames,
-      suggestionDb,
-    }),
-  })
-  function languageExtension(languageName: string): Extension | undefined {
-    const extension = extensions[languageName]
-    DEV: if (!extension) console.warn(`Unknown WidgetText syntax: ${languageName}`)
-    return extension?.value
-  }
-
+export function useLanguageSupport(syntax: ToValue<Opt<string>>): Readonly<Ref<Extension>> {
+  const languageExtension = useLanguageSupportExtensions(true)
+  if (!languageExtension) return computed(() => NULL_EXTENSION)
```

Comment:
```suggestion
  if (!languageExtension) return ref(NULL_EXTENSION)
```

With `readonly` if needed.